prefect_gcp/projects/steps.py

```diff
@@ -0,0 +1,227 @@
+"""
+Prefect project steps for code storage in and retrieval from Google Cloud Storage.
+"""
+from pathlib import Path, PurePosixPath
+from typing import Dict, Optional
+
+import google.auth
+from google.cloud.storage import Client as StorageClient
+from google.oauth2.service_account import Credentials
+from prefect.utilities.filesystem import filter_files, relative_path_to_current_platform
+from typing_extensions import TypedDict
+
+
+class PushProjectToGcsOutput(TypedDict):
+    """
+    The output of the `push_project_to_gcs` step.
+    """
+
+    bucket: str
+    folder: str
+
+
+class PullProjectFromGcsOutput(TypedDict):
+    """
+    The output of the `pull_project_from_gcs` step.
+    """
+
+    bucket: str
+    folder: str
+    directory: str
+
+
+def push_project_to_gcs(
+    bucket: str,
+    folder: str,
+    credentials: Optional[Dict] = None,
```

Comment:
Should we expose this in the recipe so that users can quickly begin using their gcs blocks?

Reply:
I can hear slackbot in my head - we should probably do this